build/pr-variables.yml

```diff
@@ -3,7 +3,8 @@ variables:
     resourceGroupRoot: 'msh-fhir-pr'
     appServicePlanName: '$(resourceGroupRoot)-$(prNumber)-asp'
     prNumber: $(system.pullRequest.pullRequestNumber)
-    DeploymentEnvironmentName: '$(resourceGroupRoot)-$(prNumber)'
+    ResourceGroupName: '$(resourceGroupRoot)-$(prNumber)'
+    DeploymentEnvironmentName: '$(build.BuildNumber)'
```

Comment:
>. [](start = 39, length = 1)

looks like `.` is not allowed for keyvault name

Reply:
One of the other variables replaces them with dashes, I think it was $(GitVersion.NuGetVersionV2)
For reference: https://github.com/GitTools/actions/blob/master/gitversion/execute/action.yml

Reply:
That is super helpful! Thank you!